{"name": "登陆成功", "status": "broken", "statusDetails": {"message": "TypeError: object of type 'NoneType' has no len()", "trace": "self = <test_case.TestCase object at 0x000001337F7354C8>, username = 'PT_LM01'\npassword = 'JYHG123.'\n\n    @pytest.mark.parametrize('username,password', [\n        (ConfYaml().get_login_user_info()['admin']['username'],\n         ConfYaml().get_login_user_info()['admin']['password'])])\n    @allure.title(\"登陆成功\")\n    def test_success_login(self, username, password):\n        while True:\n            self.driver.page_clear_verify_code()  # 清空验证码\n            # 截图（元素）并识别\n            time.sleep(1)\n            image = \"../image/{}.png\".format(time.strftime(\"%Y_%m_%d %H-%M-%S\"))\n            self.driver.page_get_iamge_element(image)\n>           self.driver.page_excute_login(username, password, self.docr.docr(image))\n\ntest_case.py:44: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\page\\login_page.py:53: in page_excute_login\n    self.page_input_verify_code(verify_code)  # 执行验证码输入\n..\\page\\login_page.py:19: in page_input_verify_code\n    self.base_input(page.verify_code, code)\n..\\base\\base.py:23: in base_input\n    el.send_keys(value)\nC:\\Program Files\\Python\\Python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py:478: in send_keys\n    {'text': \"\".join(keys_to_typing(value)),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nvalue = (None,)\n\n    def keys_to_typing(value):\n        \"\"\"Processes the values that will be typed in the element.\"\"\"\n        typing = []\n        for val in value:\n            if isinstance(val, Keys):\n                typing.append(val)\n            elif isinstance(val, int):\n                val = str(val)\n                for i in range(len(val)):\n                    typing.append(val[i])\n            else:\n>               for i in range(len(val)):\nE               TypeError: object of type 'NoneType' has no len()\n\nC:\\Program Files\\Python\\Python37\\lib\\site-packages\\selenium\\webdriver\\common\\utils.py:150: TypeError"}, "attachments": [{"name": "log", "source": "3a4a41f7-8131-4a7f-8b8c-fc305df8e42f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'PT_LM01'"}, {"name": "password", "value": "'JYHG123.'"}], "start": 1686040375583, "stop": 1686040376849, "uuid": "1f00e16d-7cc6-4938-9675-2395d0d7ed24", "historyId": "28ff920126121d61270d970a9c0cc46a", "testCaseId": "4cbc5e44c17f52326963ed5c3f8b3eb9", "fullName": "testcase.test_case.TestCase#test_success_login", "labels": [{"name": "feature", "value": "TestCase"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "lll"}, {"name": "thread", "value": "26064-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_case"}]}